dspace/config/local.cfg.EXAMPLE

```diff
@@ -85,18 +85,41 @@ db.password = dspace
 #      so this may be set to ${db.username} in most scenarios.
 db.schema = public
 
-## Connection pool parameters
+## Database Connection pool parameters
+# DSpace wraps around the Apache Commons DBCP2 component, to read about its configuration
+# see: https://commons.apache.org/proper/commons-dbcp/configuration.html
 
-# Maximum number of DB connections in pool (default = 30)
-#db.maxconnections = 30
+# Maximum number of active DB connections in pool (-1 = unlimited)
+# (default = 30)
+db.maxconnections = 30
 
-# Maximum time to wait before giving up if all connections in pool are busy (milliseconds)
+# Maximum time to wait before giving up if all connections in pool are busy (milliseconds), (-1 = unlimited)
 # (default = 5000ms or 5 seconds)
-#db.maxwait = 5000
+db.maxwait = 5000
```

Comment:
Same here, doesn't need to be uncommented. This can remain commented out (as it already was)